src/utils/registerCommands.ts

```diff
@@ -0,0 +1,20 @@
+import { fetch } from "./fetch";
+import { commandTypes } from "../typeDefinitions/register.types";
+
+export async function registerCommands(
+  url: string,
+  discordBotToken: string,
+  commands: Array<commandTypes>
+) {
+  const response = await fetch(url, {
+    headers: {
+      "Content-Type": "application/json",
+      Authorization: `Bot ${discordBotToken}`,
+    },
+    method: "PUT",
+    body: JSON.stringify(commands),
+  });
+
+  if (response.ok) console.log("Registered all commands");
+  else console.error("Error Registering Commands");
```

Comment:
Would it be good to handle async calls via try/catch

Reply:
Updated the PR